n3fit/src/n3fit/scripts/n3fit_exec.py

```diff
@@ -159,6 +159,8 @@ def from_yaml(cls, o, *args, **kwargs):
             N3FIT_FIXED_CONFIG['use_thcovmat_in_sampling'] = thconfig.get(
                 'use_thcovmat_in_sampling', True
             )
+        # TODO: this whole function is upside down?
+        N3FIT_FIXED_CONFIG["allow_legacy_names"] = file_content.get("allow_legacy_names", False)
```

Comment:
```suggestion
```

Is changing the value from the runcard not specifically what you want to avoid?

Reply:
The value in the runcard should take precedence imho.
But we can remove the option to use it (so if someone wants to be evil they need to really hack the code)